integreat_cms/static/src/js/tinymce-plugins/mediacenter/plugin.js

```diff
@@ -19,6 +24,11 @@
                             const imageEl = document.createElement("img");
                             imageEl.src = file.url;
                             imageEl.alt = file.altText;
+                            if (imageEl.alt === "" || imageEl.alt === null) {
+                                linkEl.addEventListener("focus", anchorFocusBlur);
```

Comment:
I am not quite sure what this means. As far as I understand, this tries to immediately unfocus the link when it has been focused?
If that is the case, then shouldn't setting the tabindex attribute be enough?
Also, this would only the html code of the tinymce editor. But since this event listener is not stored in the html code, it will not exist anymore when a client downloads the html page.

Reply:
@david-venhoff thanks for detailed clarification. Now it makes sense. To your question why not only setting tabindex, tabindex makes unfocusable to the users of tab key, but if someone uses mouse or touchscreen, AFAIK tabindex will not work in those cases.

Reply:
@david-venhoff we do not need any changes here anymore then? If so, I will remove the changes on this file

Reply:
Yes, I think have the functionality once at `content_utils.py` is sufficient